TAScheduler/models.py

```diff
@@ -81,8 +81,8 @@ def toDict(self):  # for display
 
 class Lecture(models.Model):
     section = models.ForeignKey(Section, on_delete=models.CASCADE, null=False)
-    instructor = models.ForeignKey(Instructor, unique=True, on_delete=models.SET_NULL, null=True)
-    ta = models.ForeignKey(TA, unique=True, on_delete=models.SET_NULL,
+    instructor = models.ForeignKey(Instructor, on_delete=models.SET_NULL, null=True)
+    ta = models.ForeignKey(TA, on_delete=models.SET_NULL,
```

Comment:
just curious why we don't want the user fields to be "unique." I'm guessing it's because it might be redundant if we're making every user unique.